woodwork/serializers/parquet_serializer.py

```diff
@@ -25,7 +38,89 @@ def serialize(self, dataframe, profile_name, **kwargs):
         self.kwargs["engine"] = "pyarrow"
         return super().serialize(dataframe, profile_name, **kwargs)
 
-    def write_dataframe(self):
-        file = self._get_filename()
-        dataframe = clean_latlong(self.dataframe)
-        dataframe.to_parquet(file, **self.kwargs)
+    def save_to_local_path(self):
+        """Serialize data and typing information to a local directory. Overrides method on base class
+        due to different serialiation flow"""
+        if self.data_subdirectory:
+            location = os.path.join(self.write_path, self.data_subdirectory)
+            os.makedirs(location, exist_ok=True)
+        else:
+            os.makedirs(self.write_path, exist_ok=True)
+        self._create_pyarrow_table()
+        self._generate_parquet_metadata()
+        self._save_parquet_table_to_disk()
+
+    def _create_pyarrow_table(self):
```

Comment:
thanks for changing the name! I think this makes it clearer what the function does!